usaon_vta_survey/templates/response/data_products.html

```diff
@@ -41,12 +63,46 @@ <h3>Data products</h3>
         {% endfor %}
       </table>
     </div>
+
+    {% if sankey_series == [] %}
+      <div>Please input at least one dataproduct-to-observing systems relationship to display diagram</div>
+    {% else %}
+      <figure class="highcharts-figure">
+        <div id="highcharts-container" />
+      </figure>
+    {% endif %}
```

Comment:
Maybe also a macro for the sankey HTML (including the script tag? Maybe they can't be right next to each-other...)? Whether 1 macro or 2 macros, I think these should be macros :)